src/tellor_disputables/data.py

```diff
@@ -141,44 +115,37 @@ async def is_disputable(reported_val: float, query_id: str, conf_threshold: floa
         return None
 
 
-@dataclass
-class NewReport:
-    """NewReport event."""
-
-    tx_hash: str
-    eastern_time: str
-    chain_id: int
-    link: str
-    query_type: str
-    value: float
-    asset: str
-    currency: str
-    query_id: str
-    disputable: Optional[bool]
-    status_str: str
-
-
-def timestamp_to_eastern(timestamp: int) -> str:
-    """Convert a timestamp to Eastern Standard time."""
-    est = tz.gettz("EST")
-    dt = datetime.fromtimestamp(timestamp).astimezone(est)
-
-    return str(dt)
-
+async def chain_events(
+    cfg: TelliotConfig, chain_addy: dict[int, str], topics: list[str], wait: int
+) -> List[List[tuple[int, Any]]]:
+    """"""
+    events_loop = []
+    for chain_id, address in chain_addy.items():
+        try:
+            endpoint = cfg.endpoints.find(chain_id=chain_id)[0]
+            if endpoint.url.endswith("{INFURA_API_KEY}"):
```

Comment:
we removed this .env key, we instead now import keys from telliot's `endpoints.yaml`

Reply:
oh wait, i see what you did. this is actually very clever!

Reply:
great catch